sandy-city-utah/active-directory-api
# fix(migrations): custom migration script

### What Changed?
  - remove excess logic in knexfile
  - remove config loading logs
  - create migration script file that handles migrations (and fixes types)
  - modify the npm scripts slightly to target custom migration script

### Issues Addressed:
  - P21312-22

### Issue Testing:
  - Manual Developer Testing

### Test Results:
1. From the command line, run `npm run migration:add -- some-migration-name`
1. See that it creates the migration in the right place
1. Run `npm run migration:run`
1. See that it runs the migration successfully and outputs the current version to the console
1. Run `npm run migration:rollback`
1. See that it rolls back the test migration successfully and outputs the current version to the console.
1. Make sure you remove the new, empty migration from the folder
